src/applications/caregivers/config/form.js

```diff
@@ -227,4 +228,7 @@ const formConfig = {
   },
 };
 
+/* TODO Need to change editModeOnReviewPage for document upload to true 
```

Comment:
Could not fix document upload review page request due to VSP bug reference ticket for details.